doc/onboarding_guide.rst

```diff
@@ -0,0 +1,38 @@
+Onboarding Best Practices
+=========================
+
+You'll have a dev contact on the CLI team. Early and frequent communication with this contact is essential to ensuring a smooth onboarding.
+
+Suggested timeline:
+
+- **First Week:** Reach out to your contact on the CLI team. Set up a short 30 minute Skype call to discuss initial questions.
```

Comment:
Maybe 'initial kickoff' and 'initial review' instead of 'first week' and 'second week', and add an iterate as needed step.

Reply:
> If your team is not familiar with Python, we will send a copy of [Python for Kids](https://www.amazon.com/Python-Kids-Playful-Introduction-Programming/dp/1593274076) over.